compaction/compaction.cc

```diff
@@ -1242,15 +1251,29 @@ class scrub_compaction final : public regular_compaction {
         flat_mutation_reader_v2 _reader;
         mutation_fragment_stream_validator _validator;
         bool _skip_to_next_partition = false;
+        std::optional<compaction_task_stats_ptr> _cts;
```

Comment:
Since `compaction_task_stats_ptr` is a `lw_shared_ptr` it doesn't need to be optional.
It's default-constructed to have no payload..